virttest/env_process.py

```diff
@@ -360,7 +360,8 @@ def postprocess_image(test, params, image_name, vm_process_status=None):
         # with a new backup. i.e. assume pre-existing image/backup
         # would not be usable after this test succeeds. The best
         # example for this is when 'unattended_install' is run.
-        if params.get("backup_image", "no") == "yes":
+        if (params.get("backup_image_after_testing_passed", "no") == "yes" and
```

Comment:
we can still use the same param`backup_image` and additional check of `test_passed` is good.

Reply:
`backup_image` implies both test passed or failed.
If you want to save an image when test for debug, you can use `save_image`.

Reply:
exactly, so lets default `backup_image` when `test_passed` = True, why add new param, and unness change in all testcases too.

Reply:
@sathnaga  That is a old version of this patch, i update it to the current version as i think it is better to add the usage scenario to the param name, same with `backup_image_before_testing`, `restore_image_after_testing`.  To make it more clearly when use this param. Thanks!

Reply:
make sense, pls make sure take care of  all instances this param usage.

Reply:
@sathnaga thanks for reminder,  do not worry, i checked tp-qemu. tp-libvirt and also tp-spice, will cover all ^^